README.md

```diff
@@ -166,6 +167,7 @@ The following tools can be used to visualize genomic data or for constructing cu
 * __[scribl](https://github.com/chmille4/Scribl)__ - JavaScript library for drawing canvas-based gene diagrams. The [Homepage](http://chmille4.github.io/Scribl/) has examples.
 * __[DNAism](https://github.com/drio/dnaism)__ - Horizon chart D3-based JavaScript library for DNA data.
 * __[Circleator](https://github.com/jonathancrabtree/Circleator)__ - Flexible circular visualization of genome-associated data with BioPerl and SVG.
+* __[BioJS](https://github.com/biojs)__ - BioJS is a library of over hundred JavaScript components enabling you to visualize and process data using current web technologies.
```

Comment:
Should this be changed to the landing page rather than the GitHub organization page? It's not very clear as to how to start using this library from the organization page. It makes more sense to link to the landing page https://biojs.net/.